assets/files/etc/NetworkManager/dispatcher.d/pre-up.d/z-baremetalupdatescript

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+IF=ens4
```

Comment:
This should use the logic in /usr/local/bin/keepalived.sh to determine the right interface.

Reply:
Ya, on the dev2 baremetal env the interfaces are named differently

> 2: eno1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000\    link/ether e4:43:4b:52:79:54 brd ff:ff:ff:ff:ff:ff
> 3: eno2: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 1500 qdisc mq state UP mode DEFAULT group default qlen 1000\    link/ether e4:43:4b:52:79:55 brd ff:ff:ff:ff:ff:ff

Reply:
changed the logic so that we rely on the interface which has default gw